src/ActionMenu2.tsx

```diff
@@ -95,7 +95,9 @@ const Overlay: React.FC<MenuOverlayProps> = ({children, ...overlayProps}) => {
     'anchorRef'
   >
 
-  const {containerRef, openWithFocus} = useMenuInitialFocus(open, onOpen)
+  const containerRef = React.createRef<HTMLDivElement>()
+  const {openWithFocus} = useMenuInitialFocus(open, onOpen, containerRef)
+  useMenuTypeaheadFocus(open, containerRef)
```

Comment:
Changed both of these hooks to accept a ref if it's passed (or create a new one if it's not passed)